src/pages/index.js

```diff
@@ -91,36 +125,37 @@ profileEditButton.addEventListener("click", () => {
     configSettings
   );
 });
-profileExitButton.addEventListener("click", () => {
-  closeModal(profileModal);
-});
 profileForm.addEventListener("submit", (evt) => {
-  evt.preventDefault();
-  closeModal(profileModal);
-  profileName.textContent = profilNameInput.value;
-  profileDesc.textContent = profileDescInput.value;
+  handleFormSubmit(evt, () => {
+    return api
+      .editUserInfo({
+        name: profilNameInput.value,
+        about: profileDescInput.value,
+      })
+      .then((data) => {
+        profileName.textContent = data.name;
+        profileDesc.textContent = data.about;
+        closeModal(profileModal);
+      })
+      .catch(console.error);
+  });
 });
 
 // add card events
 addCardOpenButton.addEventListener("click", () => {
   openModal(addCardModal);
 });
-addCardExitButton.addEventListener("click", () => {
-  closeModal(addCardModal);
-});
 addCardForm.addEventListener("submit", (evt) => {
-  evt.preventDefault();
-  closeModal(addCardModal);
-  cardList.prepend(
-    getCardElement({ link: addCardFormLink.value, name: addCardFormName.value })
-  );
-  evt.target.reset();
-  disableButton(addCardSubmitButton, configSettings);
-});
-
-// preivew events
-previewExitButton.addEventListener("click", () => {
-  closeModal(previewModal);
+  handleFormSubmit(evt, () => {
+    return api
+      .addCard({ link: addCardFormLink.value, name: addCardFormName.value })
+      .then((res) => {
+        closeModal(addCardModal);
+        cardList.prepend(getCardElement(res));
+        disableButton(addCardSubmitButton, configSettings);
+      })
+      .catch(console.error);
```

Comment:
* `catch` is not needed if you use `handleFormSubmit`, because it contains `catch` already